vsts/publish.sh

```diff
@@ -80,14 +80,7 @@ else
   prerelease=$(echo "$ARTIFACT_PATH"/*.zip | rev | cut -d/ -f1 | rev)
   zip_filename="$(echo $FRAMEWORKS_ZIP_FILENAME | cut -d. -f1)"
   commit_hash="$(echo $prerelease | sed 's/'$zip_filename'-[[:digit:]]\{1,\}.[[:digit:]]\{1,\}.[[:digit:]]\{1,\}-[[:digit:]]\{1,\}+\(.\{40\}\)\.zip.*/\1/1')"
-
-  ### Temporarily remove tvOS framework from binary
-  unzip $ARTIFACT_PATH/$prerelease
-  rm -rf $FRAMEWORKS_ZIP_FOLDER/tvOS
-  zip -r $FRAMEWORKS_ZIP_FILENAME $FRAMEWORKS_ZIP_FOLDER/
-
-### Once we support tvOS, we just need to rename the file.
-#  mv $prerelease $FRAMEWORKS_ZIP_FILENAME
+  mv $prerelease $FRAMEWORKS_ZIP_FILENAME
```

Comment:
This won't work. You should add an artifact PATH which is changed in master branch. I need to hold this merge until we have master -> develop after release.

Reply:
#1670 will unblock to make a change for this one.